Update/hima_003_04.txt

```diff
@@ -839,7 +839,7 @@ void main()
 //　去るのみの自分に何か危害が加えられる可能性は、常識的に考えてとても低い＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　去るのみの自分に何か危害が加えられる可能性は、常識的に考えてとても低い。",
-		   NULL, "With the only thing left for me to do was leaving, the chance that I would come to harm was logically fairly low.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "With the only thing left for me to do being to leave, the chance that I would come to harm was logically fairly low.", GetGlobalFlag(GLinemodeSp));
```

Comment:
```suggestion
		   NULL, "With leaving being the only thing left for me to do, the chance that I would come to harm was logically fairly low.", GetGlobalFlag(GLinemodeSp));
```

Would re-wording the sentence like this work instead?

Reply:
Yep, I do think this flows a bit better! :)